package.json

```
@@ -65,34 +65,34 @@
     "infusion": "4.6.0",
     "markdown-it": "13.0.1",
     "npm-run-all": "4.1.5",
-    "nunjucks": "3.2.3",
-    "postcss": "8.4.21",
+    "nunjucks": "3.2.4",
+    "postcss": "8.4.24",
     "stylelint-order": "6.0.3"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.4.4",
-    "@commitlint/config-conventional": "17.4.4",
-    "@sindresorhus/slugify": "2.2.0",
-    "backstopjs": "6.1.4",
+    "@commitlint/cli": "17.6.5",
+    "@commitlint/config-conventional": "17.6.5",
+    "@sindresorhus/slugify": "2.2.1",
+    "backstopjs": "6.2.1",
     "cross-env": "7.0.3",
     "eleventy-plugin-error-overlay": "1.0.0",
-    "eslint": "8.35.0",
+    "eslint": "8.41.0",
     "eslint-plugin-react": "7.32.2",
     "estrella": "1.4.1",
     "html-minifier": "4.0.0",
     "husky": "8.0.3",
     "laravel-mix": "6.0.49",
-    "linkedom": "0.14.23",
-    "lint-staged": "13.1.2",
+    "linkedom": "0.14.26",
+    "lint-staged": "13.2.2",
     "move-file": "2.1.0",
     "netlify-cms-proxy-server": "1.3.24",
     "resolve-url-loader": "5.0.0",
-    "rimraf": "4.3.1",
-    "sass": "1.58.3",
-    "sass-loader": "13.2.0",
+    "rimraf": "5.0.1",
+    "sass": "1.62.1",
+    "sass-loader": "13.3.1",
     "start-server-and-test": "2.0.0",
-    "stylelint": "14.16.1",
+    "stylelint": "15.6.2",
     "stylelint-config-fluid": "github:fluid-project/stylelint-config-fluid#8b6f7964ceb21acf690efb07b28fdda589ecfdd2",
-    "stylelint-config-standard-scss": "6.1.0"
```

Adjusting these dependencies resolves the remaining CI error.

```suggestion
    "stylelint": "14.16.1",
    "stylelint-config-fluid": "1.0.0",
    "stylelint-config-standard-scss": "6.1.0"
```

---

Hey @greatislander, after these adjustments, I am still getting js lint error from infusion-uio.js. Is this happening to you as well?

---

Oh, but the checks on CI has passed, I am seeing the issues on local env.

---

@chosww Can you reset your local environment? Unstage any local changes, run `npm ci` and try to lint again?